README.md

```diff
@@ -13,13 +13,15 @@ provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) fo
 - [Installation](#installation)
   - [Prerequisites](#prerequisites)
   - [Installing Minishift](#installing-minishift)
-    - [Installing via Homebrew](#installing-via-homebrew)
+    - [Installing with Homebrew](#installing-with-homebrew)
 - [Quickstart](#quickstart)
   - [Starting Minishift](#starting-minishift)
   - [Deploying a sample application](#deploying-a-sample-application)
   - [Reusing the Docker daemon](#reusing-the-docker-daemon)
 - [Documentation](#documentation)
 - [Known Issues](#known-issues)
+  - [Bugs](#bugs)
+  - [Unsupported features](#unsupported-features)
```

Comment:
My 2 cents, the Known Issues section need not be a part of the Readme at all, especially now, with 'Bugs' being made into a sub-heading which shows in the TOC. It's like a user, right at the onset is being told, here is a product and here are its bugs. I do not say we need to hide them but we do not need to prominently display them either.
We could have a seperate doc with known issues, like here: https://github.com/sorccu/cufon/wiki/Known-bugs-and-issues or https://github.com/dotnet/core/tree/master/cli. 
We could put a line with a link in the Community section and also in the Contributing guidelines saying, before you raise issues please have a look at this doc which lists the known issues, or such.